src/celeritas/em/interactor/MuBremsstrahlungInteractor.hh

```diff
@@ -80,21 +81,13 @@ class MuBremsstrahlungInteractor
     ParticleTrackView const& particle_;
     // Differential cross section calculator
     MuBremsDiffXsCalculator calc_dcs_;
-    // Ratio of gamma production cutoff to minimum energy cutoff
-    real_type xmin_;
-    // Ratio of incident energy to gamma production cutoff
-    real_type xmax_;
+    // Distribution to sample energy
+    ReciprocalDistribution<> sample_energy_;
     // Envelope distribution for rejection sampling of gamma energy
     real_type envelope_;
 
     //// CONSTANTS ////
 
-    //! Minimum allowed secondary cutoff energy [MeV]
-    static CELER_CONSTEXPR_FUNCTION Energy min_cutoff_energy()
-    {
-        return units::MevEnergy{9e-4};  //!< 0.9 keV
```

Comment:
Just curious why this constant was removed? Is the reciprocal distribution equivalent to what's used here to sample the energy if the production cut is larger than this value?.

Reply:
The reciprocal distribution is exactly mathematically equivalent: it turns out the min cutoff doesn't factor into the sample at all. I just refactored it a bit and simplified it so that it was the reciprocal sampling expression, and surprisingly that value simply disappeared. (The original expression had a couple of extra unnecessary `log`s.)

Reply:
gotcha, I see it now, thanks!